.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
What is this for? Keep it simple please and just put whaever is needed to build the image in `docker/` that's the only purpose of that directory :)